seacrowd/sea_datasets/vintext/vintext.py

```diff
@@ -0,0 +1,214 @@
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        zip_filepath = os.path.join(os.path.dirname(__file__), "vietnamese_original.zip")
+        if not os.path.exists(zip_filepath):
+            gdown.download(id=_GDRIVE_ID, output=zip_filepath)
+
+        data_dir = dl_manager.extract(zip_filepath)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "imagepath": os.path.join(data_dir, "vietnamese/train_images"),
+                    "labelpath": os.path.join(data_dir, "vietnamese/labels"),
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "imagepath": os.path.join(data_dir, "vietnamese/test_image"),
+                    "labelpath": os.path.join(data_dir, "vietnamese/labels"),
+                },
+            ),
+        ]
+
+    def _generate_examples(self, imagepath: Path, labelpath: Path) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        df_list = []
+
+        for image in os.listdir(imagepath):
+            image_id = int(image.split(".")[0][2:])
+            label_file = os.path.join(labelpath, f"gt_{image_id}.txt")
+            with open(label_file, "r") as f:
+                label = f.read().strip()
+            df_list.append({"id": image_id, "image_path": os.path.join(imagepath, image), "label": label})
+
+        if self.config.schema == "source":
+            for i, row in enumerate(df_list):
+                labels = [label.split(",") for label in row["label"].split("\n")]
+
+                yield i, {
+                    "id": row["id"],
+                    "image_path": row["image_path"],
+                    "annotations": [
+                        {
+                            "x1": label[0],
+                            "y1": label[1],
+                            "x2": label[2],
+                            "y2": label[3],
+                            "x3": label[4],
+                            "y3": label[5],
+                            "x4": label[6],
+                            "y4": label[7],
+                            "transcript": label[8],
+                        }
+                        for label in labels
+                    ],
+                }
+
+        elif self.config.schema == "seacrowd_imtext":
+            for i, row in enumerate(df_list):
+                labels = [label.split(",") for label in row["label"].split("\n")]
+
+                yield i, {
+                    "id": row["id"],
+                    "image_paths": [row["image_path"]],
+                    "texts": "",
```

Comment:
Let's use `None` instead since an empty string is still a string.

```suggestion
                    "texts": None,
```